akka-distributed-data/src/main/scala/akka/cluster/ddata/Replicator.scala

```diff
@@ -884,7 +884,7 @@ final class Replicator(settings: ReplicatorSettings) extends Actor with ActorLog
     OneForOneStrategy()(
       ({
         case e @ (_: DurableStore.LoadFailed | _: ActorInitializationException) if fromDurableStore ⇒
-          log.error(e, "Stopping distributed-data Replicator due to load or startup failure in durable store")
+          log.error(e.getCause, "Stopping distributed-data Replicator due to load or startup failure in durable store")
```

Comment:
I think this should still be `e` to include the right stack trace, the `cause` will be included in the logged stack trace.
To make the message even more readable we could include the message of the cause like:
```scala
log.error(e, "Stopping distributed-data Replicator due to load or startup failure in durable store, caused by: {}", if (e.getCause eq null) "" else e.getCause.getMessage)
```